_config.yml

```diff
@@ -23,107 +19,17 @@ incremental: true
 sass:
   style: :compressed
 
-# Author/Organization info to be displayed in the templates
-author:
-  name: 18F Product Chapter
-  url: https://18f.gsa.gov
-
-# Point the logo URL at a file in your repo or hosted elsewhere by your organization
-logourl: /assets/img/18f-logo.png
-logoalt: 18F logo
-
-# Navigation
-# List links that should appear in the site sidebar here
-navigation:
-- text: Welcome
-  url: index.html
-  internal: true
-- text: Product Management at 18F
-  url: product-management-at-18f/
-  internal: true
-- text: Discover the current state
```

Comment:
@hoaiandang Looks like this chunk of navigation is missing from the move. Mind re-copying?